### ISSUE TITLE
Information for redirect for already logged user

### ISSUE THREAD

**Issue by Lele991** [AnthonyNahas/ngx-auth-firebaseui#263]
```
- [ ] bug report -> please search issues before submitting
- [x] feature request
```

Hi @AnthonyNahas, thanks for the library. I tried to find from the wiki the solution to create an autoredirect when the user is already logged in the site. For example:
When the new user connect to my website his view the page for login and after the successful login his is redirected to the dashboard, when this user try to reconnect to my website they show again the page for login but his is already logged.

It's possible to create this redirect to the dashboard when they insert the default link-address for already user logged? Thanks for the support :).

**Comment by AnthonyNahas** [comment]
you need a similar mechanism like the `authGuard`... 

for the meantime, you can implement by yourself a simple guard to enable redirection when the user is logged in... I will try to add this feature to the next release

**Comment by AnthonyNahas** [comment]
@Lele991 you can either implement that by yourself or use the loggedin guard from the library like below:

```ts
import {NgxAuthFirebaseUIModule} from 'ngx-auth-firebaseui';


  NgxAuthFirebaseUIModule.forRoot(firebaseKey, firebaseAppNameFactory,
      {
        authGuardFallbackURL: 'examples/logged-out',
        authGuardLoggedInURL: 'examples/logged-in',
      }),
```

in your routing module --> 


```ts
{path: 'logged-in', component: LoggedInComponent},
{path: 'logged-out', component: LoggedOutComponent},
{path: 'guards', component: GuardsExampleComponent, canActivate : [LoggedInGuard]},
```

this available in [v3.2.0](https://www.npmjs.com/package/ngx-auth-firebaseui)

thank you 🍻
